docs/src/guidelines.md

```diff
@@ -33,6 +33,9 @@ Depth = 3
                                  "applied to a set of dimension $(dim(P))"
 ```
 
+!!! note
+    The 80 characters rule should not be enforced arbitrarily, but is rather a soft guideline.
```

Comment:
Sorry, I do not understand what this means.

Reply:
*Soft guideline* in the sense to not make it mandatory; it depends on the context.

For instance, i would break the line for arguments in a function signature ([example](https://github.com/JuliaReach/LazySets.jl/blob/master/src/Sets/Ellipsoid.jl#L234)) but not break the line if the offending line uses 83-90 characters and belongs to the body of the function.. since it is arguably *more* readable to keep the entire line.

More discussion can be fond [here](http://www.juliaopt.org/JuMP.jl/dev/style/#Line-length-1) and  [here](https://github.com/jrevels/YASGuide/pull/12).

Reply:
What confused me was the word "arbitrarily." The constant sure 80 is arbitrary (although quite common), but once it is fixed, enforcing the limit is not arbitrary. I think "in all cases"/"under any circumstances" expresses what you want.

Apart from that, we can make this change but I suggest that we still try to stick to close-to-80 limits because that is how I work with files.